### PR TITLE
Network Device Doc Adding of the HP1910 Switch

### PR DESCRIPTION
**Pull request by TomInverse**
# Description
Adding of the HP 1910 serie configuration steps.
Screenshots will be added later.

# Impacts
Must give the needed information to configure a HP1910 serie switch with PacketFence. 

# Delete branch after merge
YES

# NEWS file entries
Screenshots will be added later.

## Enhancements
Add HP 1910 Serie configuration steps,

# UPGRADE file entries
docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -3024,6 +3024,95 @@ Add the following to the previous interfaces configuration.
...
+Configure the authentication domain::
+
+INFO: Even limited, there is a command line access.
+
+* Connect to the switch using ssh,
+* Give:
+
+ _cmdline-mode on
+
+* Password: *512900*
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: ?

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -3024,6 +3024,95 @@ Add the following to the previous interfaces configuration.
...
+* Give:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Give? Or you mean "Type the following commands:"

Also, are you sure the commands start with capital letter?

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -3024,6 +3024,95 @@ Add the following to the previous interfaces configuration.
...
+* From _Max Number of Users_, give *2*,
+* Check the box _Enable Re-Authentication_,
+* Click on *Apply*,
+
+
+image::docs/images/hp1910_01.png[scaledwidth="100%",alt="802.1X Authentication"]
+
+Enable the MAC Authentication in SSH, as well::
+* Back on the SSH terminal,
+* Give:
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Same thing.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -3024,6 +3024,95 @@ Add the following to the previous interfaces configuration.
...
+ _cmdline-mode on
+
+* Password: *512900*
+* Give:
+
+ System-view
+ Mac-authentication domain YOUR_DOMAIN_NAME
+ Mac-authentication user-name-format mac-address with-hyphen
+
+* Change the _YOUR_DOMAIN_NAME_ with the one from your context.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Rephrase - "context" isn't the right term to use here. You could simply say "Change _YOUR_DOMAIN_NAME_ with the one appropriate for your environment."

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 5]
```diff
@@ -3024,6 +3024,95 @@ Add the following to the previous interfaces configuration.
...
+
+image::docs/images/hp1910_01.png[scaledwidth="100%",alt="802.1X Authentication"]
+
+Enable the MAC Authentication in SSH, as well::
+* Back on the SSH terminal,
+* Give:
+
+ Mac-authentication interface gX/X/X
+
+* Modify the interface name for your context.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Same thing about the context.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc (hunk 1 above)
Comment (TomInverse) [reply docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: done

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc (hunk 2 above)
Comment (TomInverse) [reply docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Given by the HP Support, I think, yes.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc (hunk 3 above)
Comment (TomInverse) [reply docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: done

**Pull request merged**